src/java/org/apache/cassandra/tools/nodetool/Import.java

```diff
@@ -80,6 +80,16 @@ public class Import extends NodeToolCmd
             description = "Copy data from source directories instead of moving them")
     private boolean copyData = false;
 
+    @Option(title = "fail_on_missing_index",
+            name = {"-f", "--fail-on-missing-index"},
```

Comment:
I am open to different flags here, `-f` and `-i` just makes sense to me.

Reply:
+1 to `-siv`/`--skip-index-validation`

Still thinking about the other...

Reply:
I'm fine w/ `--require-index-component` too, although I think it should be `--require-index-components`. (An SAI index is really a bag of separate components that enable related functionality, but perhaps I'm too in the weeds...)

Reply:
for `--skip-index-validation`, I do not like this one because if you notice what other flags there are, there is suffix `--no` - `--no-verify`, `--no-tokens`, `--no-invalidate-caches` - there is no `--skip...`. We might do `--no-index-validation`, that would work, but no `--skip-index-validation`.

When it comes to `--require-index-component`, I do not like the fact that this does not say what happens when they are required and they are not present. To prepend it with `--fail` is pretty obvious to me. Anyway, one can conclude that if something is required and it is not there, it will probably complain and fail eventually ... yeah, I think this could fly.